cypress/integration/CreateHwLpa.feature

```diff
@@ -74,7 +76,16 @@ Feature: Create a Health and Welfare LPA
             | address-address2| Undercliff Drive |
             | address-address3| Ventnor, Isle of Wight |
             | address-postcode| PO38 1UL |
-        Then I click "form-cancel"
-        Then I click "save-and-continue"
-        And I am taken to the when lpa starts page
-        And I see "When can the LPA be used?" in the page text
+        When I click "form-cancel"
+        And I click "save-and-continue"
+        Then I am taken to the life sustaining page
+        And I see "Who does the donor want to make decisions about life-sustaining treatment?" in the page text
+        # in this test we check CanSustainLife-0 (no option) exists, then a few lines down we actually click canSustainLife-1 (yes)
```

Comment:
Just an FYI, when I fixed the radio button focus order, the quickest thing to do was to give the first option the ID which was previously given to the enclosing fieldset. This means that the links in the error summary will move focus onto the first radio button. If I didn't do this, it would mean reworking the whole of the multiselect and error summary elements in PHP/Twig so that the error messages point to the first radio button (which doesn't have a predictable ID). The upshot is that any tests which look for radio buttons may have to use different IDs. On the other hand, I did add a data-cy attribute to every one so you don't have to find them by ID in tests. I will figure this out when I merge from master (after we've merged this PR), but just a heads-up for the future.

Reply:
Thanks for the info :)